quickstarts/microsoft.machinelearningservices/machine-learning-job-create-automl-classification-job/main.bicep

```diff
@@ -0,0 +1,69 @@
+@description('Specifies the name of the Azure Machine Learning workspace where job will be created.')
+param workspaceName string
+
+@description('Specifies the name of the Azure Machine Learning compute instance/cluster on which job will be run.')
+param computeName string
+
+@description('Specifies the name of the Azure Machine Learning experiment under which job will be created.')
+param experimentName string
+
+@description('Specifies the name of the Azure Machine Learning job to be created.')
+param jobName string
+
+@description('Specifies execution contraints for the job.')
+param limitSettings object = {
+  maxTrials: 5
+  maxConcurrentTrials: 1
+  maxCoresPerTrial: -1
+  enableEarlyTermination: true
+  timeout: 'PT10H'
+  trialTimeout: 'PT20M'
+}
+
+@description('Specifies training related configuration for the job.')
+param trainingSettings object = {
+  blockedTrainingAlgorithms: [
+    'LogisticRegression'
+  ]
+  enableModelExplainability: true
+  enableOnnxCompatibleModels: false
+  enableStackEnsemble: true
+  enableVoteEnsemble: true
+  ensembleModelDownloadTimeout: 'PT5M'
+  enableDnnTraining: false
+}
+
+var compute = resourceId('Microsoft.MachineLearningServices/workspaces/computes', workspaceName, computeName)
+
+resource jobResource 'Microsoft.MachineLearningServices/workspaces/jobs@2022-06-01-preview' = {
+  name: '${workspaceName}/${jobName}'
+  properties: {
+    jobType: 'AutoML'
+    tags: {
+      referenceNotebook: 'https://github.com/Azure/azureml-examples/blob/main/sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing'
+    }
+    experimentName: experimentName
+    computeId: compute
+    taskDetails: {
+      logVerbosity: 'Info'
+      taskType: 'Classification'
+      primaryMetric: 'Accuracy'
+      targetColumnName: 'y'
+      trainingData: {
+        uri: 'https://raw.githubusercontent.com/Azure/azureml-examples/main/sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/data/training-mltable-folder/'
```

Comment:
any artifacts needed for deployment need to be part of the sample - you cannot cross-link to another repo

Reply:
Thanks, I've made the required changes. The CI correctly does the deployment. But do we expect the user to pass the value of `_artifactsLocation`? Because if they don't (e.g. when using cli), cli fails with:

```
> az deployment group create --name DeployLocalTemplate --resource-group <rg> --template-file .\main.bicep --parameters workspaceName=<ws> computeName=<compute> experimentName=<exp> jobName=<job>

{"code": "InvalidTemplate", "message": "Deployment template validation failed: 'The template resource '_artifactsLocation' at line '36' and column '27' is not valid: The language expression property 'templateLink' doesn't exist, available properties are 'template, templateHash, parameters, mode, provisioningState'.. Please see https://aka.ms/arm-template-expressions for usage details.'.", "additionalInfo": [{"type": "TemplateViolation", "info": {"lineNumber": 36, "linePosition": 27, "path": "properties.template.parameters._artifactsLocation"}}]}
```